### ISSUE TITLE
How to change date format on the Client UI

### ISSUE THREAD

**Issue by SirwanAfifi** [Xabaril/AspNetCore.Diagnostics.HealthChecks#109]
Hi, thanks for this project, I'm wondering if I could change the date time format on the Client UI page. Do we have a setting for this?

![image](https://user-images.githubusercontent.com/5256416/53245207-5474b200-36c2-11e9-8110-dd3f698d2be0.png)

**Comment by unaizorrilla** [comment]
Hi @CarlosLanderas 

This issue can be an interesting new feature! :-)

**Comment by CarlosLanderas** [comment]
Yes we can add the desired, we can add it in the UI options 👍. I'll do it

**Comment by unaizorrilla** [comment]
I think that the feature is change the uiculture, not only the date format, right?

**Comment by CarlosLanderas** [comment]
We can do it both ways, configuring culture option with current UI culture or date format in JsonSerialization Settings so they are automatically formatted on deserialization.

Whatever you prefer

**Comment by snavarropino** [comment]
Hi guys, what about give the option to humanize dates? 
I mean, let user decide to see dates in following format:

2 hours ago
20 minutes ago
a day ago

**Comment by rahul230691** [comment]
I need help regarding the same. Is it implemented?

**Comment by wudiqiang2024** [comment]
I need help regarding the same. Is it implemented?

**Comment by wudiqiang2024** [comment]
@CarlosLanderas How to set the timespan format of DURATION column?
